client/src/shared/state/Chat.js

```diff
@@ -68,10 +68,18 @@ class Chat {
         if (result === undefined) throw new Error(`Server returned an unknown scope: ${scope}`);
     }
 
+    isAlphaNumeric(str) {
+        return str.match("^[A-Za-z0-9]+$");
```

Comment:
isAlphaNumeric does not detect messages containing white space characters as target for filtering.
`^[\w\s]+$`
would match that kind of messages

Reply:
Updated at 7007eb0